packages/cozy-app-publish/README.md

```diff
@@ -208,3 +208,21 @@ We apply the semver convention:
 - If the release brings only bug fixes, then we should create a patch version.
 - If the release brings at least one feature, then we should create a minor version.
 - If the release changes its application's permissions, we should create a minor version even if this release is for bug fixes only.
+
+### Hot Fix
+
+If you need to qucikly fix a bug in production, then you have to:
+- fetch the latest branch release 
+- create a new release branch from there (since this is a bug fix, this is a patch version)
+- fix the bug 
+- Make a beta
```

Comment:
Why is it necessary to do a beta for a hot fix?

Reply:
To be sure that we didn't break anything when fixing the bug?

Reply:
I would have assumed we already test everything before pushing :)